src/Symfony/Component/HttpKernel/Debug/FileLinkFormatter.php

```diff
@@ -16,6 +16,7 @@
 trigger_deprecation('symfony/http-kernel', '6.4', 'The "%s" class is deprecated, use "%s" instead.', FileLinkFormatter::class, ErrorHandlerFileLinkFormatter::class);
 
 class_exists(ErrorHandlerFileLinkFormatter::class);
+class_alias(ErrorHandlerFileLinkFormatter::class, FileLinkFormatter::class);
```

Comment:
the alias already exists in `ErrorHandlerFileLinkFormatter` so this should not be needed
are you checking cross version failures ? I removed the alias from 7.0, maybe that's the issue?

Reply:
tests are failing on 6.3 with 6.4 of the HttpKernel component installed: https://github.com/symfony/symfony/actions/runs/6550029688/job/17788242726#step:8:10353

Reply:
okay, but the ErrorHandler component is installed in version 7

Reply:
good point, I wasn't aware of the other class alias, `class_exists()` check added